packages/merge-styles/src/styleToClassName.ts

```diff
@@ -240,7 +240,7 @@ export function styleToRegistration(...args: IStyle[]): IRegistration | undefine
   }
 }
 
-export function applyRegistration(registration: IRegistration, classMap?: { [key: string]: string }): void {
+export function applyRegistration(registration: IRegistration): void {
```

Comment:
Even though this file is exported, it is not part of the package's public api surface. I believe this shouldn't be a breaking change.